CONTRIBUTING.md

```diff
@@ -76,9 +76,9 @@ The release process must include the following steps:
 
 ## Coding Conventions and Style Guide
 
-- The SDKs are written in Swift, however they still have to be compatible for use from Objective-C based apps.
+- The SDKs are written in Swift.
 - Favor Protocol Oriented Programming with Dependency Injection when writing any code. We're unable to create automatic mocks in Swift, so it'll be helpful for writing unit tests.
 - SwiftLint is integrated into the project. Make sure that your code does not add any SwiftLint related warning.
 - Please remove default Xcode header comments (with author, license and creation date) as they're not necessary.
 - If you're adding or modifying any part of the public interface of SDK, please also update [QuickHelp](https://developer.apple.com/library/archive/documentation/Xcode/Reference/xcode_markup_formatting_ref/SymbolDocumentation.html#//apple_ref/doc/uid/TP40016497-CH51-SW1) documentation.
-- Docs are written for both Swift and ObjC
+- Docs are written for Swift
```

Comment:
I'm not sure that the two modified bullet points make sense after removing the mention of Objective-C. Do you think they can just be removed now?

Reply:
d0c5da66930d285e2d181270b8ae5ee56fff7475

Reply:
I was also referring to "The SDKs are written in Swift."

Reply:
> I was also referring to "The SDKs are written in Swift."

be80427ea18338b69ed1a1a348b773dd9de4f352